### PR TITLE
As discussed in PR "Update to cucumber 4.5" #357

### PR DESCRIPTION
**Pull request by nhojpatrick**

### CODE REVIEW

**Review by luke-hill — APPROVED** [review]
Sure? I guess. I'll let a JVM maintainer decide if these numbers are right.

I'm guessing they point to a stable version?

**Comment by mpkorstanje** [comment]
The numbers are right!

**Pull request merged**
